packages/pattern-commands/src/listeners/PluginCommandAccepted.ts

```diff
@@ -34,7 +24,7 @@ export class CommandAcceptedListener extends Listener<typeof PatternCommandEvent
 			return duration;
 		});
 
-		result.inspectErr((error) => message.client.emit(PatternCommandEvents.CommandError, error, { ...payload, duration: -1 }));
+		result.inspectErr((error) => message.client.emit(PatternCommandEvents.CommandError, error, { ...payload, error, duration: -1 }));
```

Comment:
The error is already the first parameter of the emit, it doesn't need to also be in the object.

Reply:
Removed it from the object